ClinGen/clincoded
# Connects to #843. Criteria bar.

**Technical details:**
The criteria bar has been updated to demonstrate the following behaviors:
- White background with colored text: default status
- Gray background with colored text: "Not met" status
- Colored background with white text: "Met" status at any strength level

**Steps to test:**
1. Login and use **12345** at the variant selection interface.
2. Start new interpretation and associate the interpretation with disease if needed.
3. Expect to see the display of the criteria bar with its default behavior -- buttons with white background and colored text.
4. Try different evaluation status (e.g. "Met" or "Not Met") on different criteria and expect to see the corresponding criteria bar buttons to change to "Colored background with white text" or "Gray background with colored text" accordingly.
